src/pytest_dbfixtures/factories/postgresql.py

```diff
@@ -121,7 +121,7 @@ def drop_postgresql_database(psycopg2, user, host, port, db):
     conn = psycopg2.connect(user=user, host=host, port=port)
     conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
     cur = conn.cursor()
-    cur.execute('DROP DATABASE IF EXISTS %s' % db)
+    cur.execute('DROP DATABASE IF EXISTS %s;' % db)
```

Comment:
I don't understand why other queries work without that semicolon, but this one doesn't.